tests/test_data/projects/migration_streamlit_V2/snowflake.yml

```diff
@@ -0,0 +1,15 @@
+definition_version: 2
```

Comment:
Please add test for migrating snowflake.yml with templates. Templates should not be rendered in outcome file.

Reply:
Done